src/main/java/bio/terra/workspace/service/status/StatusService.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.workspace.service.status;
+
+import bio.terra.workspace.generated.model.SystemStatus;
+
+/**
+ * An interface for a service providing status information about this server.
+ *
+ * <p>This has two implementations in Workspace Manager. The first, ScheduledStatusService, is used
+ * for all deployments of the service. It checks all dependencies on a schedule and reports cached
+ * statuses via endpoints.
+ *
+ * <p>The second, FakeStatusService, is only used for tests and always returns an OK status with no
+ * subsystems. This is useful for preventing unexpected interactions between scheduled methods and
+ * tests.
```

Comment:
I don't think the interface is the right place for this comment. It probably won't get updated if this info changes, and the implementors can be found and have their own javadoc.
I think it's worth moving the comment about why we have a FakeStatusService to its class. I think BaseStatusService has most of what's needed in its javadoc already.